configs/view-eln.yaml

```diff
@@ -75,6 +75,15 @@ data:
   # be pulled into a RHEL-like environment.
   - python3.6
 
+  # Miro Hrončok:
+  # The older Python version might accidentally be pulled in by not yet rebuilt packages
+  - python3.8
```

Comment:
As a side note, could you please rebuild glade-libs and unbound-libs with Python 3.9?

Reply:
> As a side note, could you please rebuild glade-libs and unbound-libs with Python 3.9?

I will take care of this.